pkg/migration-tool/reader/reader.go

```diff
@@ -78,7 +80,10 @@ func (r *Read) Run(errChan chan<- error) {
 				errChan <- fmt.Errorf("remote-run run: %w", err)
 				return
 			}
-			ms := []*labels.Matcher{labels.MustNewMatcher(labels.MatchRegexp, labels.MetricName, ".*")}
+			ms := r.Config.MetricsMatchers
+			if len(ms) == 0 {
+				ms = []*labels.Matcher{labels.MustNewMatcher(labels.MatchRegexp, labels.MetricName, ".*")}
+			}
```

Comment:
If I am not wrong, this will never be the case, as the validation checks with parser & vector_selector will always prevent this. But I think this is good for a sanitary check, so we can just add a warning saying, to open an issue if this block is ever executed, as then it will be some corner case in the code. 

```suggestion
			if len(ms) == 0 {
			        log.Warn("msg", "empty matchers received. Please open an issue regarding this at https://github.com/timescale/promscale/issues")
				ms = []*labels.Matcher{labels.MustNewMatcher(labels.MatchRegexp, labels.MetricName, ".*")}
			}
```

Reply:
Done